amplfi/law/amplfi/law/base.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from amplfi.law.paths import paths
+
+
+class AmplfiDataTaskMixin:
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.image = paths().container_root / "data.sif"
+
+    def sandbox_env(self, env):
+        # get `mldatafind.law.DataTask` environment variables
```

Comment:
Can you provide a use case example here i.e. what `env`

Reply:
Yeah this is a `law` hook for mapping local environment variables into the container that the tasks are run inside. So for `mldatafind` these are things like credentials, segment database urls, etc.

Reply:
Let's add that as a docstring, otherwise its unclear.

Reply:
done